docs/kilnjs.md

```diff
@@ -6,6 +6,8 @@ sidebar_label: Kilnjs
 
 ---
 
+⚠️You must being using Clay-cli version 3.10.2 or greater to use Kiln.js⚠️
```

Comment:
Suggestion: we could add bold style to the version

⚠️You must being using Clay-cli version **3.10.2** or greater to use Kiln.js⚠️

Reply:
👍